x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -842,6 +845,8 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
           clickRuleUpdatesTab();
 
           openRuleUpdatePreview(UPDATED_CUSTOM_QUERY_INDEX_PATTERN_RULE['security-rule'].name);
+          assertSelectedPreviewTab('JSON view');
```

Comment:
Let's create reusable constants for these tab names across this file.

```
const PREVIEW_TABS = {
  OVERVIEW: 'Overview',
  JSON_VIEW: 'JSON view',
  UPDATES: 'Updates', // I know it's not used, but to have as a list of all tabs
}
```

Reply:
Addressed in a follow-up commit